Makefile

```diff
@@ -0,0 +1,3 @@
+create_venv:
```

Comment:
that's not good name anymore, probably better something like `docs`

Reply:
done